docs/adding_software/overview.md

```diff
@@ -4,17 +4,39 @@ We welcome contributions to the EESSI software stack. This page shows the proced
 
 ## Contribute a software to the EESSI software stack
 
-<p align="center">
-  <img width="100%" src="../../img/contribute_software_workflow.png" alt="EESSI contribute software">
-</p>
+<pre class="mermaid">
+%%{init: { 'theme':'forest', 'sequence': {'useMaxWidth':false} } }%%
+flowchart TB
+    I(contributor)  
+    K(reviewer)
+    A(Is there an EasyConfig for software) -->|No|B(Create an EasyConfig and contribute it to EasyBuild)
+    A --> |Yes|D(Create a PR to software-layer)
+    B --> C(Evaluate and merge pull request)
+    C --> D
+    D --> E(Review PR & trigger builds)
+    E --> F(Debug build issue if needed)
+    F --> G(Deploy tarballs to S3 bucket)
+    G --> H(Ingest tarballs in EESSI by merging staging PRs)
+     classDef blue fill:#9abcff,stroke:#333,stroke-width:2px;
+     class A,B,D,F,I blue
```

Comment:
This makes the mermaid graphs a little more interactive. people can be sent to the documentation of the related node. 
```suggestion
     class A,B,D,F,I blue
     click B "https://easybuild.io/"
     click D "../opening_pr/"
     click F "../debugging_failed_builds/"
```